es/synonyms/he.txt

```diff
@@ -146,8 +145,8 @@
 
 # ConventionLocation
 סנט פטרסבורג	פיטר
-ארצות הברית ארהב
-תורכיה  טורקיה
+ארצות הברית	ארהב
```

Comment:
What is this change? Bug? Space instead of tab?

Reply:
tab added instead whitespace